src/locale-data/en-US.json

```diff
@@ -104,5 +104,16 @@
 			"percent": "%"
 		},
 		"groupSize": 3
+	},
+	"fileSize": {
```

Comment:
You'll want to bring in the locale data for all the other locales as well. It should all be there contained in the language terms the existing formatter is using.